ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1620,6 +1621,92 @@ private class JdbcLedgerDao(
     }
   }
 
+  private val SQL_SELECT_COMMAND = SQL("""
+      |select submitted_at, ttl, success, error
+      |from participant_command_submissions
+      |where deduplication_key = {deduplicationKey}
+    """.stripMargin)
+
+  case class ParsedCommandData(
```

Comment:
Minor: this could be `private` and `final`.

Reply:
Adding private final to all similar case classes in this file.